src/miral/static_display_config.cpp

```diff
@@ -124,106 +124,113 @@ try
     if (!layouts.IsDefined() || !layouts.IsMap())
         throw mir::AbnormalExit{error_prefix + "no layouts"};
 
-    auto const layout = begin(config["layouts"]);
+    auto layout = layouts["default"];
 
-    if (layout != end(config["layouts"]))
+    if (!layout.IsDefined() || !layout.IsMap())
     {
-        Node name_node;
-        Node config_node;
+        // No 'default' but there is only one - use it
```

Comment:
I'd just error out if default layout wasn't there. But if you think that's better, WFM.

Reply:
@Saviq I have been using it as easy-to-implement behaviour that is compatible with my test files. But I can drop it now if we don't like it.